scholarly/publication_parser.py

```diff
@@ -235,22 +235,19 @@ def _scholar_pub(self, __data, publication: Publication):
             if publication['bib']['abstract'][0:8].lower() == 'abstract':
                 publication['bib']['abstract'] = publication['bib']['abstract'][9:].strip()
 
+        publication['url_scholarbib'] = _BIBCITE.format(cid, pos)
+        sclib = self.nav.publib.format(id=cid)
+        publication['url_add_sclib'] = sclib
+
         lowerlinks = databox.find('div', class_='gs_fl').find_all('a')
 
         publication["num_citations"] = 0
 
         for link in lowerlinks:
-            if (link is not None and
-                    link.get('title') is not None and
-                    'Cite' == link.get('title')):
```

Comment:
It's difficult to see why this stopped working, but I do not see any reason why the following three lines have to be inside the loop. The option to `Cite` is always present to the best of my knowledge and requires no checking. If we need to be cautious, we should be checking on `cid` and `pos`.